### ISSUE TITLE
DeprecationWarning: pkg_resources is deprecated as an API.

### ISSUE THREAD

**Issue by honnorat** [nsidc/earthaccess#419]
Use of `pkg_resources` is deprecated in favor of [importlib.resources](https://docs.python.org/3.11/library/importlib.resources.html#module-importlib.resources):

```
$ python -W always -c "import earthaccess"
/home/honnorat/usr/conda/envs/earthaccess/lib/python3.12/site-packages/earthaccess/formatters.py:4:
    DeprecationWarning: pkg_resources is deprecated as an API.
    See https://setuptools.pypa.io/en/latest/pkg_resources.html
  import pkg_resources
```
The code in   `formatters.py`:
```python
import pkg_resources

STATIC_FILES = ["css/iso_bootstrap4.0.0min.css", "css/styles.css"]

def _load_static_files() -> List[str]:
    """Load styles"""
    return [
        pkg_resources.resource_string("earthaccess", fname).decode("utf8")
        for fname in STATIC_FILES
    ]
```
could be replaced by:
```python
import importlib.resources

def _load_static_files() -> List[str]:
    """Load styles"""
    return [
        (
            importlib.resources.files("earthaccess") / fname
        ).read_text("utf8")
        for fname in STATIC_FILES
    ]
```

**Comment by mfisher87** [comment]
Thank you for the very detailed report! Would you be comfortable opening a pull request?

**Comment by chuckwondo** [comment]
Fixed by #497. (See also comment on #452 regarding `importlib_resources`, rather than `importlib.resources`).
